docs/src/concepts/index_ivfpq.md

```diff
@@ -0,0 +1,78 @@
+# Understanding LanceDB's IVF-PQ index
+
+An index is a data structure that represents data in a way that makes it more efficient to search and retrieve. Using an index is faster, but less accurate than kNN or brute force search because, in essence, the index is a lossy representation of the data.
+
+LanceDB is fundamentally different from other vector databases in that it is built on top of [Lance](https://github.com/lancedb/lance), an open-source columnar data format designed for performant ML workloads and fast random access. Due to the design of Lance, LanceDB's indexing philosophy adopts a primarily *disk-based* indexing philosophy.
+
+## IVF-PQ
+
+IVF-PQ is a composite index that combines inverted file index (IVF) and product quantization (PQ). The implementation in LanceDB provides several parameters to fine-tune the index's size, query throughput, latency and recall, which are described later in this section.
+
+### Product quantization
+
+Quantization is a compression technique used to reduce the dimensionality of an embedding to speed up search. 
+
+![](../assets/ivfpq_quantization.webp)
```

Comment:
It's unclear what this image is showing. What is the yellow versus blue? What is `S` and `D`?

Reply:
You're right, it's unclear. The figure here doesn't really help illustrate the point we're trying to make, so I've removed this and updated the PQ figure per your feedback below.